src/lib/dialog/dialog.spec.ts

```diff
@@ -150,6 +196,25 @@ describe('MdDialog', () => {
     expect(overlayContainerElement.querySelector('md-dialog-container')).toBeFalsy();
   });
 
+  it('should notify the user that overlay backdrop was clicked', () => {
+    let dialogRef = dialog.open(PizzaMsg, {
+      viewContainerRef: testViewContainerRef
+    });
+
+    viewContainerFixture.detectChanges();
+
+    let backdrop = overlayContainerElement.querySelector('.cdk-overlay-backdrop') as HTMLElement;
+
+    let clicked = false;
```

Comment:
Instead of keeping a flag like this, you should use a spy:

```ts
let clickedSpy = jasmine.createSpy('clicked callback');

dialogRef.backdropClicked.subscribe(clickedSpy);

expect(clickedSpy).toHaveBeenCalled();
```